torchci/pages/api/issue/[label].ts

```diff
@@ -1,15 +1,21 @@
 import fetchIssuesByLabel from "lib/fetchIssuesByLabel";
 import { IssueData } from "lib/types";
 import { NextApiRequest, NextApiResponse } from "next";
+import zlib from "zlib";
 
 export type IssueLabelApiResponse = IssueData[];
 
 export default async function handler(
   req: NextApiRequest,
-  res: NextApiResponse<IssueLabelApiResponse>
+  res: NextApiResponse
 ) {
   return res
     .status(200)
     .setHeader("Cache-Control", "s-maxage=60")
-    .json(await fetchIssuesByLabel(req.query.label as string));
+    .setHeader("Content-Encoding", "gzip")
+    .send(
+      zlib.gzipSync(
+        JSON.stringify(await fetchIssuesByLabel(req.query.label as string))
+      )
+    );
```

Comment:
can recipients parse this new response format without needing any change in code?

Reply:
Yes, it works with no additional changes